AdminDashboard/BusinessLogicOrchestrators/LoginOrchestrator/LoginOrchestrator.cs

```diff
@@ -0,0 +1,108 @@
+﻿using AdminDashboard.BusinessLogicOrchestrators.LoginOrchestrator;
+using AdminDashboard.Exceptions;
+using AdminDashboard.Main.Configurations;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace GravitationalTest.BusinessOrchestrators.Users
+{
+    public class LoginOrchestrator : ILoginOrchestrator
+    {
```

Comment:
Here is the logic that authenticates the user, and if she is authorized, a JWT token will be generated. Some things to note in regards to the problem statement:
1. It is allowed to hard-code the Username (in the future, would probably store it in the database)
2. The password is hashed and stored using the NET standard (pbkdf2)
3. It's okay to hard-code the bearer token in appsettings.json